src/components/mailing/IFXMailButton.vue

```diff
@@ -0,0 +1,106 @@
+<script>
+// Speed dial control for sending items to to:, cc:, or bcc: fields on IFXMailingCompose
+// v-model should be bound to the "selected" list.  Check for length of input determines enabled state
+
+export default {
+  name: 'IFXMailButton',
+  props: {
+    value: {
+      type: String,
+      required: true,
+    },
+    disabled: {
+      type: Boolean,
+      required: false,
+      default: false,
+    },
+    toolTip: {
+      type: String,
+      required: false,
+      default: 'Compose message with selected',
+    },
+    direction: {
+      type: String,
+      required: false,
+      default: 'bottom',
+    },
+    icon: {
+      type: String,
+      required: false,
+      default: 'mdi-email-send-outline'
+    },
+    color: {
+      type: String,
+      required: false,
+      default: 'green',
+    }
+  },
+  data() {
+    return {
+      mailFab: false
+    }
+  },
+  methods: {
+    setRecipientField(field) {
+      this.$emit('input', field)
+    }
+  }
+}
+</script>
+<template>
+  <v-tooltip top>
+    <template v-slot:activator="{ on, attrs }">
+      <div v-on="on">
+        <v-speed-dial
+          :direction="direction"
+          v-model="mailFab"
+          v-bind="attrs"
+        >
+          <template v-slot:activator>
+            <v-btn
+              v-model="mailFab"
+              small
+              :color="color"
+              fab
+              :disabled="disabled"
+            >
+              <v-icon color="white" v-if="mailFab">
+                mdi-close
+              </v-icon>
+              <v-icon color="white" v-else>
+                {{ icon }}
+              </v-icon>
+            </v-btn>
+          </template>
+          <v-btn
+            xSmall
+            fab
+            color="#A4F323"
```

Comment:
Would it be better to use the Vuetify colors here?

Reply:
May be a bad idea, but going with a single color scheme for the "email button" everywhere.

Reply:
I meant using the Vuetify colors such as `blue lighten-1` rather than a specific RGB color.